README.md

```diff
@@ -1,41 +1,134 @@
-# Open Source Project Template
+﻿# Cartography Refactor
 
-This repository contains a template to seed a repository for an Open Source
-project.
-
-## How to use this template
-
-1. Check out this repository
-2. Delete the `.git` folder
-3. Git init this repository and start working on your project!
-4. Prior to submitting your request for publication, make sure to review the
-   [Open Source guidelines for publications](https://nventive.visualstudio.com/Internal/_wiki/wikis/Internal_wiki?wikiVersion=GBwikiMaster&pagePath=%2FOpen%20Source%2FPublishing&pageId=7120).
-
-## Features (to keep as-is, configure or remove)
-- [Mergify](https://mergify.io/) is configured. You can edit or remove [.mergify.yml](/.mergify.yml).
-- [allcontributors](https://allcontributors.org/) is configured. It helps adding contributors to the README.
-- [dependabot](https://dependabot.com/) is configured. This bot will open pull requests automatically to update nuget dependencies. This one could be annoying, feel free to remove the [.dependabot](/.dependabot) folder.
-
-The following is the template for the final README.md file:
-
----
-
-# Project Title
-
-{Project tag line}
-
-{Small description of the purpose of the project}
+Doing a complete refactor of cartography module.  
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+### Samples
+- Clone project.
+- For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP)
+
+### DynamicMap
+- Add Cartography.DynamicMap NuGet package to your project.
+- In your ViewModel :
+```
+using Cartography.DynamicMap
+```
+- Implement IDynamicMapComponent to your ViewModel
+- Set Initial Value to your ViewPort.
+- Add in your Page
+#### UWP
+```
+<win:Grid>
```

Comment:
We need to keep using win: at the moment?

Reply:
Right now yes, but we want to change it.